libs/agora/genes/src/lib/components/gene-search/gene-search.component.ts

```diff
@@ -187,7 +187,7 @@ export class GeneSearchComponent implements AfterViewInit, OnDestroy {
 
   checkClickIsInsideComponent(event: Event) {
     // if clicked element is not part of this component, hide gene results
-    if (!this.root.nativeElement.contains(event.target)) {
+    if (!this.root?.nativeElement?.contains(event.target)) {
```

Comment:
Navigation to the home component (by links from another component) result in a TS error: TypeError: Cannot read properties of undefined (reading 'nativeElement')